Add MobilePay (#338)
vippsas/vipps-recurring-api: vipps-recurring-api.md

### Charge descriptions

> When charges are shown to users in Vipps or MobilePay, they will have a title, and a

Reviewer: There, I fixed it! 😅

```suggestion
When charges are shown to users in the app, they will have a title, and a
```
